Unity/Assets/GameSystems/Unity/Components/Movement/Movement2DTransform.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using UnityEngine;
+
+public class Movement2DTransform : MonoBehaviour
+{
+    [SerializeField] float speedX;
```

Comment:
Need a private after the [serializeField], as mentioned here - https://github.com/AdolfoPSakata/UnityTraining-2/pull/1